d/Dockerfile

```diff
@@ -3,33 +3,23 @@ FROM debian:bookworm AS build
 WORKDIR /usr/src/app
 
 RUN apt-get -qq update
-RUN apt-get -qy install build-essential wget openssl ldc dub libssl-dev zlib1g-dev
+RUN apt-get -qy install build-essential openssl ldc dub libssl-dev zlib1g-dev llvm-dev
 
 {{#files}}
   COPY '{{source}}' '{{target}}'
 {{/files}}
 
 ENV DC=ldc2
-RUN dub build -b release --compiler=ldc2 --verbose
+RUN dub build -b release-nobounds --compiler=ldc2
```

Comment:
I though this not recommended
https://github.com/the-benchmarker/web-frameworks/pull/6488#discussion_r1253091679

Reply:
I think it depends on type of the system. If it is big web application with a lot of logic - probably no. If it is simple micro-service where owner would like to see max performance - probably fine.
And everyone else are still using it.

Reply:
In this comment, it was specific about D, seems fine in `nim` for example.

Also, the logic will grow later

Reply:
Please remove this flag,  and I will update results with this PR

Reply:
In last PR move back to 'release'